library/CMakeLists.txt

```diff
@@ -100,14 +104,17 @@ endif(WIN32)
 
 if(USE_PKCS11_HELPER_LIBRARY)
     set(libs ${libs} pkcs11-helper)
+    list(APPEND PKGCONFIG_LIBS "-lpkcs11-helper")
```

Comment:
You need to add libpkcs11-helper-1 to PKGCONFIG_REQUIRES instead for this one

[jamrial@ArchVM ~]$ pkg-config --libs libpkcs11-helper-1
-lpthread -ldl -lcrypto -lgnutls -lssl3 -lsmime3 -lnss3 -lnssutil3 -lplds4 -lplc4 -lnspr4 -lpkcs11-helper

With shared builds linking to -lpkcs11-helper alone wouldn't be an issue, but with static ones it will fail to link to all the above dependencies.